team-norms/supported_software.md

```diff
@@ -0,0 +1,25 @@
+# Supported Software
+
+Where the day-to-day running of an application needs to transition to our support team, there are specific considerations you must take when making technology choices.
+
+As an organisation we're keen to embrace new and diverse technology where there's a strong commercial justification.
+
+When you're considering a major technology choice that isn't documented here, before starting a production implementation you must seek agreement from the support team that they're able to provide BAU support.
+
+The decision to self-host a third party application must involve the support team. You can expect to be encouraged to seek SaaS alternatives.
+
+Where the application is transitioning to a customer support team for ongoing support and maintenance, you're strongly advised to consult with the customer team on any major technology decisions.
+
+## Currently Supported Software
+
+- Ruby on Rails
```

Comment:
Something doesn't feel right about this. Software isn't automatically maintainable because it uses ruby on rails.

Reply:
Sure, there's more to maintainable software than technology choice, but what's being described here are technologies the support team are currently capable in offering BAU support for. A team decides C# is the the right tool for their job? They need to have a conversation with the support team before making that decision as it's placing an additional burden on the support team that they'll need to plan for.

Reply:
You may consider this PR if you're looking at suggesting other capabilities software must provide to be maintainable: https://github.com/madetech/handbook/pull/139/files